### ISSUE TITLE
Issue on page /supervised/classify.html

### ISSUE THREAD

**Issue by MatildeRFonseca** [statisticalbiotechnology/dsbook#6]
Under the section _6.2.3. Cross-entropy loss_, there is the following sentence:

"The probably most used loss function for classification cross-entropy loss for classification tasks."

This sentence feels a bit off to me, perhaps I think it misses an "is" before "cross-entropy" and also classification appears twice.

**Comment by percolator** [comment]
This is right. I upodated accordingly.

Closed bu e31320d8b430ee24ca9a66b987c93233266ba97e
